Example/WalletApp/PresentationLayer/Wallet/Notifications/Models/SubscriptionsViewModel.swift

```diff
@@ -45,6 +45,7 @@ struct SubscriptionsViewModel: Identifiable {
     }
 
     var hasMessage: Bool {
-        return messagesCount != 0
+        /* return messagesCount != 0 Badge disabled */
+        return false
```

Comment:
what is the outcome here? No notification badge?

Reply:
It's disabled because we currently cant show real notifications count on initial load. Messages loading with pagination and we don't know real amount